{{cookiecutter.package_name}}/src/{{cookiecutter.package_name}}/tools/make_artifacts.py

```diff
@@ -0,0 +1,194 @@
+"""Main application functions for building artifacts.
```

Comment:
Put a stub here for build artifacts and the implementation as a separate PR.  We should talk about things that go here vs. maybe things we put into vivarium_cluster_tools and import as library code.

Reply:
Stubbed.